src/client/features/view/components/graph/contextMenu.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('lodash');
+
+const defaults = {
+  menuRadius: 80,
+  fillColor: 'rgba(0, 0, 0, 0.75)',
+  activeFillColor: 'rgba(1, 105, 217, 0.75)',
+  activePadding: 10,
+  indicatorSize: 12,
+  separatorWidth: 3,
+  minSpotlightRadius: 12,
+  maxSpotlightRadius: 12,
+  openMenuEvents: 'cxttapstart taphold',
+  itemColor: 'white',
+  itemTextShadowColor: 'transparent',
+  zIndex: 9999,
+  atMouse: false
+};
+
+const expandCollapse = generateCommand('settings_overscan', 'Expand/Collapse', ele => {
+  ele.emit('hideTooltips');
+  ele.emit('expandCollapse');
+});
+const getInfo = generateCommand('info_outline', 'More Info', ele =>  ele.emit('showTooltip'));
+
+//Create a cxt menu command object
+function generateCommand(icon, title, selectFunction) {
+  return {
+    content: '<i class="material-icons cxt-icon">' + icon + '</i> <div class="cxt-info">' + title + '</div>',
+    select: selectFunction
+  };
+}
+
```

Comment:
I think it makes sense to move this file to common/cy/events/  Is there a particular reason you think it should go in components/graph?

Reply:
The actual file could be moved. As for the binding of events, I found that binding before elements were added to the graph caused errors with the extension which is why I bound he events in /graph/index.js